### DISCUSSION TITLE
Po Table: diferenças entre colunas com e sem tipo number

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1800] [Tire suas dúvidas] [answered]
Estou com uma dúvida ao usar a Po Table definindo a coluna como tipo number.

Ao não especificar este tipo, o título da coluna e seu conteúdo ficam alinhados corretamente, mas ao especificar o tipo number, o título da coluna e o seu conteúdo ficam divergentes. A coluna fica alinhada a direita enquanto o conteúdo fica a esquerda.

Como posso fazer para alinhar o conteúdo também a direita quanto a coluna for do tipo number?

Sem number:
![sem_number](https://github.com/po-ui/po-angular/assets/46979335/84f19481-31a7-4b48-93fe-d62d1cbf3cfe)

Com number:
![com_number](https://github.com/po-ui/po-angular/assets/46979335/271c82d2-c986-4e70-9038-0a7017be1a22)

Exemplo: https://stackblitz.com/edit/po-ui-3bcxuy?file=src%2Fapp%2Fapp.component.html

**Comment by francisco7comm** [comment, marked as answer]
Oi @gtrabach , passei por essa situação recentemente.

Vi inspecionando o elemento que uma classe CSS é adicionada para o campo do tipo number a fim de alinhá-lo à direita, mas realmente não surte efeito.

Para contornar esse problema eu criei um código CSS específico no meu projeto:

```
po-table {
    .po-table-column+.po-table-column-right {
        .po-table-column-cell {
          display: contents;
        }
    }
}
```

Usei o valor contents, mas outros como inline e inline-flex também atendem. A minha ideia foi tirar o `display: block` de padrão, e atendeu.

**Reply by gtrabach** [reply]
Ei @francisco7comm. Obrigado pela resposta, estava dando uma olhada nas issues e aparentemente este comportamento é um bug da nova versão: https://github.com/po-ui/po-angular/labels/DTHFUI-7668

Realmente havia achado estranho, pois em versões anteriores não acontecia.

**Comment by anliben** [comment]
Bom Dia Senhores!

Estamos movendo uma issue para nosso backlog, em breve iremos corrigir este comportamento.
Attr.

**Comment by anliben** [comment]
Boa Tarde Senhores!

Esta atividade já esta nativa no componente, siga nosso changelog: https://github.com/po-ui/po-angular/blob/master/CHANGELOG.md

attr.
